person_capture/gui_app.py

```
@@ -19053,24 +19151,33 @@ def _collect_cfg(self) -> SessionConfig:
             cfg.hdr_crop_format = self.hdr_crop_format_combo.currentText().lower()
         except Exception:
             cfg.hdr_crop_format = "avif"
-        cfg.hdr_sdr_output_format = str(getattr(self.cfg, "hdr_sdr_output_format", "png") or "png").lower()
+        try:
+            cfg.hdr_sdr_output_format = str(
+                self.hdr_sdr_output_format_combo.currentData() or "png"
+            ).lower()
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)